src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation/Internal/CompilationOptions.cs

```diff
@@ -15,6 +16,7 @@ public class CompilationOptions
         public static readonly string PublicSignTemplate = "--public-sign";
         public static readonly string ApplicationNameTemplate = "--application-name";
         public static readonly string OutputPathTemplate = "--output-path";
+        public static readonly string ViewsToCompileTemplate = "--file";
```

Comment:
We really don't expect anybody outside of the build target to invoke the binary directly now. The arguments name really don't matter much now.